scripts/check_formatting

```diff
@@ -44,7 +46,9 @@ find scripts -type f -exec shellcheck {} +
 # Exit codes:
 #  - 1 there are files to be formatted.
 #  - 0 everything looks fine.
-./third_party/run-clang-format/run-clang-format.py -r oak examples
+./third_party/run-clang-format/run-clang-format.py -r \
+    --exclude '*/node_modules' \
+    oak examples
```

Comment:
Out of interest, what kind of things live in a `node_modules/` directory?  The `run-clang-format.py` scripts should only hit [particular file extensions](https://github.com/project-oak/oak/blob/master/third_party/run-clang-format/run-clang-format.py#L27) &ndash; are there some files that match?

Reply:
Theoretically whichever files are included in the packages that deps ship. In our case we use the `protobufjs` dep, which stores a few `.proto` files in node_modules, which clang-formats checks. ([see example of this](https://pantheon.corp.google.com/cloud-build/builds/3765d695-b83c-4451-8f32-ee8b40dd51bb;step=17?project=oak-ci))

For some of the other formatting we check (eg `.cc`), we currently don't have relevant files in our `node_modules`. I still added the `node_modules` excludes for these though, since atm we always whitelist the same set of folders. 

I could be more granular here, but a general "don't lint node_modules" seems reasonable.

Reply:
SGTM